README.md

```diff
@@ -1,75 +1,111 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
 
+# Microverse
+### Prerequisites
 
-## Built With
+- Basic html, css and medium Javascript know how.
+- Git installation
+- A text editor 
+- A terminal
+- A web browser to view output e.g Google Chrome
+- An IDE e.g Visual studio code
 
-- Major languages
-- Frameworks
-- Technologies used
+#### I have implement the following interactions:
 
-## Live Demo (if available)
+Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
+Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
+Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
 
-[Live Demo Link](https://livedemo.com)
```

Comment:
As required by the previous reviewer, kindly deploy your website on GitHub pages and add the link to a `Live Demo` section in your README file

Reply:
@emmiiorji I have done that before @[github-pages](https://github.com/apps/github-pages) github-pages bot temporarily deployed to github-pages [10 hours ago ](https://github.com/GraceAbike/Capstone/pull/3#event-7450848620)Inactive
because of live demo the project was not approve

Reply:
Okay. Sorry about that.